packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -84,19 +121,32 @@ export const styles = (theme) => ({
       fontSize: 18,
     },
   },
-  /* Styles applied to the label element. */
-  label: {
+  [`& .${treeItemClasses.label}`]: {
     width: '100%',
     paddingLeft: 4,
     position: 'relative',
     ...theme.typography.body1,
   },
+}));
+
+const TreeItemTransition = experimentalStyled(
```

Comment:
Even if it's less clear, it's more consistent

```suggestion
const TreeItemGroup = experimentalStyled(
```